### PR TITLE
Add --version | -v support to markoc.

### PR DESCRIPTION
**Pull request by austinkelleher**

### CODE REVIEW

**Comment by austinkelleher** [comment]
Also changes `--vdom -v` to `--vdom -V`.

File: bin/markoc.js [hunk 1]
```diff
@@ -64,9 +65,13 @@ var args = require('argly').createParser({
             type: 'string[]',
             description: 'Additional directories to add to the Node.js module search path'
         },
-        '--vdom -v': {
+        '--vdom -V': {
             type: 'boolean',
             description: 'VDOM output'
+        },
+        '--version -vs': {
```
Comment (patrick-steele-idem) [inline bin/markoc.js]: Should be `--version -v`, right?

File: bin/markoc.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ var resolveFrom = require('resolve-from');
 
 // Try to use the Marko compiler installed with the project
 var markoCompilerPath;
+const resolvedMarkoPackageVersion = require('../package.json').version;
```
Comment (patrick-steele-idem) [inline bin/markoc.js]: I would resolve the version based on:

```js
var pkPath = resolveFrom(process.cwd(), 'marko/package.json');
// ...
```

**Comment by patrick-steele-idem** [comment]
Thanks for working on this!

**Pull request merged**